tests/base/boring_model.py

```diff
@@ -103,6 +103,7 @@ def training_epoch_end(self, outputs) -> None:
     def validation_step(self, batch, batch_idx):
         output = self.layer(batch)
         loss = self.loss(batch, output)
+        self.log('early_stop_on', loss, on_step=False, on_epoch=True)
```

Comment:
@ydcjeff the point of boring model is to be boring... it shouldn't log or anything else. it will make testing super difficult otherwise

Reply:
the solution is to adjust the tests in this case if you want to use the boring model. NOT change the boring model

Reply:
Okay, I will fix it tmr.